src/ontology/Makefile

```diff
@@ -1002,3 +1002,21 @@ bg-clean:
 
 succeed:
 	echo make succeeded
+	
+
+###### NICO MIGRATION: overwriting unsuccessful goals!
+
+../sparql/construct-ecs-from-xrefs.sparql:
+	echo "STRONG WARNING STRONG WARNING make $@ is skipped!" && touch $@
+	
+d2p.pro:
+	echo "STRONG WARNING STRONG WARNING make $@ is skipped!" && touch $@
+	
+reports/semantic-xref-pairs.tsv:
+	echo "STRONG WARNING STRONG WARNING make $@ is skipped!" && touch $@
+	
+imports/ncbitaxon_import.owl:
```

Comment:
but we need to be able to make this!!!

Reply:
We will, I am on it.

Reply:
Fixed.. Keeping the max memory very low actually helped (probably hit some docker max im my case because I had other stuff running).